Gopkg.toml

```diff
@@ -22,16 +22,18 @@
 
 [[constraint]]
   name = "k8s.io/api"
-  version = "kubernetes-1.8.5"
-
-[[constraint]]
-  name = "k8s.io/apiextensions-apiserver"
-  version = "kubernetes-1.8.5"
+  version = "kubernetes-1.9.3"
 
 [[constraint]]
   name = "k8s.io/apimachinery"
-  version = "kubernetes-1.8.5"
+  version = "kubernetes-1.9.3"
 
 [[constraint]]
   name = "k8s.io/client-go"
-  version = "kubernetes-1.8.5"
+  version = "kubernetes-1.9.3"
+
+
+
+[[override]]
```

Comment:
Already approved, but then noticed another PR for legacy branch and a note there on this. Should this be also changed to `[[constraint]]` as in that case?

Reply:
I removed the override and the additional dependency. We let it open now and let the package manager resolve what fits.